pkg/sriov/sriov.go

```diff
@@ -255,6 +223,36 @@ func (s *sriovManager) ApplyVFConfig(conf *types.NetConf) error {
 		}
 	}
 
+	// Set link guid
+	if conf.GUID != "" {
+		if len(conf.GUID) < 22 || conf.GUID == "00:00:00:00:00:00:00:00" {
```

Comment:
I think is better to create method to validate GUID by regex

Reply:
added